python/ray/tests/test_advanced_3.py

```diff
@@ -641,7 +664,7 @@ def test_gpu_info_parsing():
     """
     constraints_dict = resource_spec._constraints_from_gpu_info(info_string)
     expected_dict = {
-        "{}V100".format(ray_constants.RESOURCE_CONSTRAINT_PREFIX): 1
+        "{}V100".format(ray_constants.RESOURCE_CONSTRAINT_PREFIX): 1,
```

Comment:
Can you use fstring instead? We dropped 3.5 support, so we can use fstring.

Reply:
Oh awesome, I didn't know that y'all dropped 3.5 support! I'm the biggest fstring advocate you'll ever meet, I'll get on that.